### PR TITLE
add some data to applications

### PR DESCRIPTION
**Pull request by Kesigomon**
Add some data to applications.
https://discord.com/developers/docs/resources/application#application-object-application-structure

### CODE REVIEW

File: oauth2.go [hunk 1]
```diff
@@ -55,6 +55,15 @@ type Application struct {
 	Owner               *User     `json:"owner"`
 	Bot                 *User     `json:"bot"`
 	Team                *Team     `json:"team"`
+	VerifyKey           string    `json:"verify_key"`
+	RPCOrigins          *[]string `json:"rpc_origins,omitempty"`
```
Comment (CarsonHoffman) [inline oauth2.go:59]: Is having a pointer to the slice really necessary here? The field is not nullable, and I don't think that we really need to differentiate between present and not present—I don't think that the field would be sent if the array were empty.

Comment (Kesigomon) [reply oauth2.go]: Fixed!

File: oauth2.go [hunk 2]
```diff
@@ -55,6 +55,15 @@ type Application struct {
 	Owner               *User     `json:"owner"`
 	Bot                 *User     `json:"bot"`
 	Team                *Team     `json:"team"`
+	VerifyKey           string    `json:"verify_key"`
+	RPCOrigins          []string  `json:"rpc_origins,omitempty"`
+	TeamsOfServiceURL   string    `json:"teams_of_service_url,omitempty"`
+	PrivacyPolicyURL    string    `json:"privacy_policy_url,omitempty"`
+	Summary             string    `json:"summary"`
+	GuildID             string    `json:"guild_id,omitempty"`
+	PrimarySkuID        string    `json:"primary_sku_id,omitempty"`
```
Comment (CarsonHoffman) [inline oauth2.go:64]: A bit of a nit, but see: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

Although it looks a bit weird, `PrimarySKUID` is the way to go here. I know it's small, but given that all this change does is add fields, we might as well name them right. 🙂

**Comment by FedorLap2006** [comment]
Merged in by 70e829694d9938f5c0fe87e05fad138ca6c408b0

**Pull request closed without merging**